plugins/fortinet_fortipam.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "fortinet fortipam"
+  authors [
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "FortiPAM provides tightly controlled privileged access to the most sensitive resources within an organization. "
+  website "https://www.fortinet.com/products/fortipam"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /FortiPAM/,
```

Comment:
this won't give you some FPs?

Reply:
For this one I was able to find a more accurate fingerprint, but for the others nothing.